.pre-commit-config.yaml

```diff
@@ -7,6 +7,7 @@ files: |
         src/Mod/AddonManager|
         src/Tools|
         tests/src
```

Comment:
You need a vertical pipe at the end of the previous line.

Reply:
Thanks for noticing.